core/silkworm/common/util.cpp

```diff
@@ -231,4 +226,18 @@ size_t prefix_length(ByteView a, ByteView b) {
     }
     return len;
 }
+
+std::vector<std::string> split(std::string source, std::string delimiter) {
```

Comment:
I know it's trivial, but could you please add a test case?

Reply:
Yeh right. Which also made emerge a couple of cases I did not think of.
Thank you